test/test-color-scheme.rb

```diff
@@ -45,12 +45,14 @@ class TestFor8Colors < self
     def setup
       @original_term, ENV["TERM"] = ENV["TERM"], nil
       @original_color_term, ENV["COLORTERM"] = ENV["COLORTERM"], nil
+      @original_vte_version, ENV["VTE_VERSION"] = ENV["VTE_VERSION"], nil
```

Comment:
Why is this needed?

Reply:
This is needed because otherwise `guess_available_colors_from_vte_version_env` will override the result of `available_colors` whenever `VTE_VERSION` is set.

Reply:
I see.